### ISSUE TITLE
Decorated classes trigger a warning

### ISSUE THREAD

**Issue by adamdelman** [Intsights/flake8-intsights#3]
The following code triggers incorrect warnings: 
I052 main class definition should start with a  "class Name("
I053 each base class should be seperated by a new-line
```python
@decorator
class DecoratedClass(
    ClassBase,
):
    pass
```
